dataclients/kubernetes/eastwest.go

```diff
@@ -16,7 +16,7 @@ func createEastWestRouteIng(eastWestDomainRegexpPostfix, name, ns string, r *esk
 		return nil
 	}
 	ewR := *r
-	ewR.HostRegexps = []string{"^" + name + "[.]" + ns + eastWestDomainRegexpPostfix + "$"}
+	ewR.HostRegexps = []string{"^" + name + "[.]" + ns + eastWestDomainRegexpPostfix + "(:[0-9]+)?$"}
```

Comment:
As far as I can see, there is no test covering this case.

Reply:
I added some basic unit tests for this method but I also found some tests in https://github.com/zalando/skipper/blob/master/dataclients/kubernetes/kube_test.go#L3034 and the tests in the kubernetes testdata files.

Reply:
Ideally we could pass `o.KubernetesEastWestDomain` instead of `eastWestDomainRegexpPostfix` and reuse `generateHostRegexp`:
```go
ewR.HostRegexps = []string{generateHostRegexp(name + "." + ns + "." + eastWestDomain)}
```
such that every place reuses the same utility function. It would be easier then to navigate and extend in the future.

Also here 
https://github.com/zalando/skipper/blob/3da61f0422ff1233f840514aec709072e396419f/dataclients/kubernetes/ingress.go#L394
the postfix is not adjusted with port capture.

Reply:
I tried to do this refactor but some tests related to the routeId generation are failing. I will need to go deeper.